starwars/src/pages/Dashboard.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+    text-align: center;
+}
```

Comment:
legyen üres sor a fileok végén
ha a VS code-od nem engedi, guglizd ki, milyen beállítás van átállítva a kódszerkesztődben